enlightenment_assistant_next-20160608.json

```diff
@@ -0,0 +1,17 @@
+{
+  "tags": [
+    "enlightenment_assistant_next"
+  ],
+  "area": [
+    {
+      "xpos": 457,
+      "ypos": 723,
+      "width": 111,
+      "height": 27,
+      "type": "match"
+    }
+  ],
+  "properties": [
+    "workaround"

```

Comment:
why set workaround here? without a better needle name nobody but you will know why this is workaround. If this is because of a known and recorded bug please name the needle accordingly as described on https://progress.opensuse.org/projects/openqatests/wiki/#applying-quotworkaroundquot-needles, e.g. enlightenment_assistant_next-boo123456-20160608